src/content/store-and-form-setup.md

```diff
@@ -26,27 +38,46 @@ export interface IBioSummary {
 
 ---
 
+## Reduce, reuse...
+
+Plan any other forms you want to add to your form state
+
+```ts
+export interface IForm {
+  character: ICharacter; // state.form.character
+  equipment: IEquipment; // state.form.equipment
+}
+
+export interface IEquipment {
+  weaponName?: string;
+  weaponType: string;
+  armorType: string;
+}
+```
+
+---
+
 ## Setting up actions
```

Comment:
Hmm, I'm trying to think of something that would work here but not be too long or confusing 🤔 I don't quite get the first one with `you don't wanna be surprised`, though... I wonder if we can do a pun with combat actions? or maybe somewhere else something about rolling for initiative.